test/test_logging_interceptor.rb

```diff
@@ -67,6 +67,19 @@ def test_logging_interceptor_logs_customer_id
     assert_includes(sio.read, "CID: #{customer_id}")
   end
 
+  def test_logging_interceptor_uses_resource_name_if_customer_id_is_not_set
+    resource_name = "123"
+    li.request_response(
+      request: make_request(customer_id: nil),
+      call: make_fake_call,
+      method: :doesnt_matter,
+    ) do
+    end
+
+    sio.rewind
+    assert_includes(sio.read, "CID: #{resource_name}")
+  end
```

Comment:
This test won't work. A bit unsure how to test the existing behaviour, or that this fix actually works because I'm not sure what operation responds to `resource_name`, but doesn't respond to `customer_id`.

I only noticed this bug while looking through logs for recently and noticed the discrepancy :)